projects/aca-content/src/lib/components/search/search-save/dialog/save-search-dialog.component.html

```diff
@@ -0,0 +1,50 @@
+<h2 class="aca-save-search-dialog__header" mat-dialog-title>{{"APP.BROWSE.SEARCH.SAVE_SEARCH.MODAL_HEADER" | translate}}</h2>
+
+<mat-dialog-content>
+  <form [formGroup]="form" (submit)="submit()">
+    <mat-form-field class="aca-save-search-dialog__form-field">
+      <mat-label>{{ 'APP.BROWSE.SEARCH.SAVE_SEARCH.NAME_LABEL' | translate }}</mat-label>
+      <input
+        id="aca-save-search-dialog-name-input"
+        [attr.aria-label]="'APP.BROWSE.SEARCH.SAVE_SEARCH.NAME_LABEL' | translate"
+        matInput
+        required
+        [formControlName]="'name'"
+        adf-auto-focus
+      />
+
+      <mat-hint *ngIf="form.controls['name'].dirty">
```

Comment:
Instead of mat-hint, you can probably use mat-error, since the content inside is only going to be displayed in cases of errors